cfgov/jinja2/v1/events/_paginated.html

```diff
@@ -23,7 +23,7 @@
             Older
             <span class="btn_icon__right cf-icon cf-icon-right"></span>
         </a>
-        <form class="pagination_form js-validate_pagination" action="#pagination_content">
```

Comment:
I wasn't sure, I just cleared out anything that included the js pagination hooks.